packages/sanity/test/cli/graphql/__snapshots__/gen3.test.ts.snap

```diff
@@ -1631,6 +1631,10 @@ exports[`GraphQL - Generation 3 > Should be able to generate graphql schema 1`]
           "fieldName": "cdrFieldNamed",
           "type": "CdrPersonReference",
         },
+        {
+          "fieldName": "productReference",
+          "type": "GlobalDocumentReference",
```

Comment:
afaict `GlobalDocumentReference` isn't declared anywhere? 🤔

Reply:
I think its a new core type. 

@skogsmaskin its too lay to remodel GDRs as just objects right? This was why assist blew up too right? And why auto updating studios with assist are gonna be in a bad way.